push_manifest.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+IMAGE_NAME=${1:-}
+IMAGE_TAG=${2:-latest}
+DOCKER_REPOSITORY=${3:-cilium}
+DOCKER_REGISTRY=${4:-}
+
+
+IMAGE_ARCH=("amd64" "arm64")
+
+if [ -z "${IMAGE_NAME}" ]
+then
+  echo "Please specify a image name!"
+  echo -e "\nUsage::\n\tpush_manifest.sh IMAGE_NAME [IMAGE_TAG] [DOCKER_REPOSITORY] [DOCKER_REGISTRY]"
+  echo -e "\nExample::\n\tpush_manifest.sh cilium-runtime latest"
+  exit 1
+fi
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+for arch in "${IMAGE_ARCH[@]}"	
+do
+	docker push ${DOCKER_REGISTRY}${DOCKER_REPOSITORY}/${IMAGE_NAME}:${IMAGE_TAG}-${arch}
```

Comment:
These are relying on Docker, but in the main makefile we use the environment variable `$CONTAINER_ENGINE` to allow developers to choose other engines. Would you update these to ensure they use the same consistent engine?

Reply:
Done